deploy/infrastructure/modules/terraform-google-dss/README.md

```diff
@@ -0,0 +1,70 @@
+# terraform-google-dss
+
+This terraform module creates a Kubernetes cluster in Google Cloud Engine and generates 
+the tanka files to deploy a DSS instance.
+
+## Getting started
+
+### Prerequisites
+Download & install the following tools to your workstation:
+
+1. Install [terraform](https://developer.hashicorp.com/terraform/downloads).
+2. Install tools from [Prerequisites](../../../../build/README.md)
+3. Install provider specific tools:
+    1. [Google Cloud Engine](./README.md#google-cloud-engine)
+    
+#### Google Cloud Engine
+
+1. Install and initialize [Google Cloud CLI](https://cloud.google.com/sdk/docs/install-sdk).
+    1. Confirm successful installation with `gcloud version`.
+2. Check that the DSS project is correctly selected: gcloud config list project
+    1. Set another one if needed using: `gcloud config set project $GOOGLE_PROJECT_NAME`
+3. Enable the following API using [Google Cloud CLI](https://cloud.google.com/endpoints/docs/openapi/enable-api#gcloud):
+    1. `container.googleapis.com`
+    2. If you want to manage DNS entries with terraform: `dns.googleapis.com`
+4. Install the auth plugin to connect to kubernetes: `gcloud components install gke-gcloud-auth-plugin`
+5. Run `gcloud auth application-default login` to generate credentials to call Google Cloud Platform APIs.
+
+### Deployment of the Kubernetes cluster
+
+1. Create a new folder in `/deploy/infrastructure/personal/` named for instance `terraform-google-dss-dev`.
+2. Copy main.tf, output.tf and variables.tf to the new folder. (Note that the modules can be added to existing projects)
+3. Copy `terraform.dev.example.tfvars` or `terraform.example.tfvars`, rename to `terraform.tfvars` and set the variables according to your environment.
```

Comment:
I removed the example file containing the instructions based on your comment about the default fields. Indeed, putting instructions inside this file was possibly carrying misinformation over time and limiting our ability to update instructions. I moved the variables description to a Markdown file instead and only kept the dev file for the moment. I propose that we look into consolidating examples when we will have multiple clouds supported. Is that ok with you?

Reply:
That sounds good, but I don't think I can see this commit either; the latest I can see is is 210cf527882200a23c2feab4c45bebe88c931f43

Reply:
Apologies, please, see latest version pushed now.

Reply:
Personally, I would still split the "set the variables according to your environment" out as its own step because it involves a relatively large amount of time/effort, but that's a small nit

Reply:
Oups, I missed it. Thank you. Please see the last commits.

Reply:
Looks good :)